src/passf5.f90

```diff
@@ -0,0 +1,86 @@
+      subroutine passf5(Ido,l1,Cc,Ch,Wa1,Wa2,Wa3,Wa4)
+      use fftpack_kind
+      implicit none
+      real(rk) :: Cc , Ch , ci2 , ci3 , ci4 , ci5 , cr2 , cr3 , &
+                  cr4 , cr5 , di2 , di3 , di4 , di5 , dr2 , dr3 , &
+                  dr4 , dr5
+      real(rk) :: ti2 , ti3 , ti4 , ti5 , tr2 , tr3, &
+                  tr4 , tr5 , Wa1 , Wa2 , Wa3 , Wa4
+      integer :: i , Ido , k , l1
+      dimension Cc(Ido,5,l1) , Ch(Ido,l1,5) , Wa1(*) , Wa2(*) , Wa3(*), &
+                Wa4(*)
+      real(rk),parameter :: pi = acos(-1.0_rk)
+      real(rk),parameter :: tr11 =  cos(2.0_rk * pi / 5.0_rk)
+      real(rk),parameter :: ti11 = -sin(2.0_rk * pi / 5.0_rk)
+      real(rk),parameter :: tr12 = -cos(4.0_rk * pi / 5.0_rk)
```

Comment:
```suggestion
      real(rk),parameter :: tr12 =  cos(4.0_rk * pi / 5.0_rk)
```
It seems that the original author wrote an extra minus sign in the comment, haha.

Here should be `tr12 = cos(..) = -0.8090169943749474241D0`

```Fortran
!> Demo
program main
      use iso_fortran_env, only: rk => real64
      real(rk),parameter :: pi = acos(-1.0_rk)
      real(rk),parameter :: tr11 =  cos(2.0_rk * pi / 5.0_rk)
      real(rk),parameter :: ti11 = -sin(2.0_rk * pi / 5.0_rk)
      real(rk),parameter :: tr12 =  cos(4.0_rk * pi / 5.0_rk)
      real(rk),parameter :: ti12 = -sin(4.0_rk * pi / 5.0_rk)

      print "(F23.20)", tr11 ! 0.30901699437494745126
      print "(F23.20)", ti11 !-0.95105651629515353118
      print "(F23.20)", tr12 !-0.80901699437494734024
      print "(F23.20)", ti12 !-0.58778525229247324813

end program main
```

Reply:
The existing unit tests do not seem to be comprehensive enough, I will add a longer data later, perform FFT, create a new unit test.